athena-neptune/docs/neptune-connector-setup/README.md

```diff
@@ -3,10 +3,10 @@
 To deploy the Amazon Athena Neptune connector, we will need the following pre-requisite information:
 1)	SpillBucket – You can either use an existing S3 bucket or create a new one to be used by the connector to store spill over results for Athena to consume.
 
-2)	NeptuneClusterEndpoint – You can get this information from the Neptune console and copying the cluster “Writer” endpoint information.
```

Comment:
This can not be merged; I believe this is wrong; Master seems to have different names for these. WE got to make sure which is the recent one; the panama version or the master.

Reply:
I believe this is just a documentation change, so no functionality concerns. Looking at the two branches, the naming is the same for both yamls. Under the hood, functionality is same for glue connection and legacy. Let's revert this on the documentation to match the CFN naming.

Reply:
can't accept this change; let's revert it back.